po/hi.po

```diff
@@ -0,0 +1,864 @@
+# Hindi translation for avahi.
+# Copyright (C) 2024 avahi's COPYRIGHT HOLDER
+# This file is distributed under the same license as the avahi package.
+# Scrambled 777 <weblate.scrambled777@simplelogin.com>, 2024.
+# Scrambled777 <weblate.scrambled777@simplelogin.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: avahi master\n"
```

Comment:
Could you change it to `Avahi` to make it consistent with the other `po` files?

Reply:
`"Project-Id-Version: Avahi master\n"`

Like this?

Reply:
`"Project-Id-Version: Avahi\n"`

I should have suggested a change instead of that ambiguous comment. Sorry!

Reply:
> I should have suggested a change instead of that ambiguous comment. Sorry!

No worries.

Amended `po` file. 

Anyway, I got pot file from [here](https://l10n.gnome.org/vertimus/avahi/master/po/hi/).
Is it possible to have updated POT file with the modifications you mentioned? It will avoid further mistakes in future.

Reply:
> Is it possible to have updated POT file with the modifications you mentioned?

I'm not sure who has access to https://l10n.gnome.org/vertimus/avahi/master/po/hi/ and how it gets updated so I'm afraid I can't help here.